manager/integration/tests/test_ha.py

```diff
@@ -2426,3 +2426,94 @@ def test_replica_failure_during_attaching(settings_reset, client, core_api, volu
     del update_disks["extra-disk"]
     node.diskUpdate(disks=update_disks)
     common.wait_for_disk_update(client, node.name, 1)
+
+
+@pytest.mark.skip(reason="TODO") # NOQA
+def test_engine_image_missing_on_some_nodes():
+    """
+    This e2e test follows the manual test steps at:
+    https://github.com/longhorn/longhorn/issues/2081#issuecomment-783747541
+
+    Steps:
+
+    Preparation:
+    1. Set up a backup store
+    2. Let's name the 3 nodes: node-1, node-2, node-3
+
+    Case 1: Test volume operations when engine image DaemonSet is miss
+    scheduled
+    1. Create a volume, vol-1, of 3 replicas
+    2. Create another volume, vol-2, of 3 replicas
+    3. Taint node-1 with the taint: key=value:NoSchedule
+    4. Verify that all functions (attach, snapshot, backup,
```

Comment:
snapshot and backup for detached volumes? Do you mean recurring jobs?

Reply:
Sorry for not being clear. It shoud be `Verify that we can attach, take snapshot, take a backup,
       detach, then expand vol-1` 
Fixed